_posts/2019-04-09-sr2019-tlc-crowned-champion.md

```diff
@@ -0,0 +1,113 @@
+| 1st Place             | [The Ladies' College](http://www.ladiescollege.com/)
+| 2nd Place             | [Hills Road Sixth Form College](http://www.hillsroad.ac.uk/)
+| 3rd Place             | [Hampton School](https://hamptonschool.org.uk/) and [Lady Eleanor Holles School](https://www.lehs.org.uk/)
+| <span title="The rookie team who achieve the highest place in the league">Rookie Award</span>          | [Eltham College](https://www.elthamcollege.london/)
+| <span title="The team that displays the most extraordinary ingenuity in the design of their robot">Committee Award</span>       | [South Wilts Grammar School](https://www.swgs.wilts.sch.uk/)
+| <span title="The team that presents their robot and themselves in what is judged to be the most outstanding way">Robot and Team Image</span>  | [Haberdashers' Aske's Boys' School](https://www.habsboys.org.uk/)
+| <span title="The team that is judged to have the best online presence using the hashtag #srobo19">Online Presence</span>       | [Collyer's](https://www.collyers.ac.uk/) ([Twitter](https://twitter.com/CollyersRobots))
+
+We always like to see new teams joining the competition, and Eltham College were still able to pull their weight against our seasoned competitors. Their robot picked up a stack of tokens and used tank tracks as a method of climbing the volcano.
+
+The Committee Award is especially important to us. As engineers, we are looking for the most simple and elegant solution to every challenge, and Team SWI from South Wilts Grammar School managed to do exactly that. Their robot was designed to take a single token into the highest scoring zone, the Caldera, by using [large wheels to climb the volcano](https://twitter.com/studentrobotics/status/1114534154283175936). Over the course of the competition weekend, they made notes of other teams' strategies and were constantly evolving their own to stay competitive. They also quickly and cleverly changed their approach to overcome technical difficulties midway through the competition.
+
+We love to see teams dressing up to compliment their robot theme, and this year Haberdashers' Aske's Boys' School won the Robot and Team Image award with their Hawaiian theme. This was complimented with inflatable palm trees and a 3D paper pineapple on top of their robot.
+
+Lastly, we like to see the robot-building process over the course of the year, and Collyer's were keeping us up-to-date on [Twitter](https://twitter.com/CollyersRobots) all the way from Kickstart.
+
+For full details on all the awards, please see the [rulebook](/docs/resources/2019/rulebook.pdf).
+
+Of course, this competition would not be possible without the support of [The Motorola Solutions Foundation](https://www.motorolasolutions.com/en_us/about/company-overview/corporate-responsibility/motorola-solutions-foundation.html), our sponsor for this year, who enabled us to provide a fun and educational opportunity for students from a range of engineering disciplines. 
```

Comment:
nit: git tells me there's trailing whitespace on this line.
```suggestion
Of course, this competition would not be possible without the support of [The Motorola Solutions Foundation](https://www.motorolasolutions.com/en_us/about/company-overview/corporate-responsibility/motorola-solutions-foundation.html), our sponsor for this year, who enabled us to provide a fun and educational opportunity for students from a range of engineering disciplines.
```

Reply:
https://github.com/srobo/website/pull/148/commits/8c9b7a4defb4c55299fdb75dc2b846a8bb8c242c